app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,77 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.dto.UserData;
+import com.codesoom.assignment.exception.UserNotFoundException;
+import com.codesoom.assignment.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    /**
+     * id 값과 일치하는 User를 리턴합니다.
+     *
+     * @param id 조회할 User의 id
+     * @return User의 id와 일치하는 User
+     * @throws UserNotFoundException User의 id와 일치하는 User가 없을 때 던집니다.
```

Comment:
`@throws`, `@return` 같은 것에서는 마침표를 빼고 작성해 보세요. 

```suggestion
     * @throws UserNotFoundException User의 id와 일치하는 User가 없을 경우
```

Reply:
네 !! 알겠습니다.